README.md

```diff
@@ -16,97 +20,59 @@ This playbook aims to install Trento components and the belonging third parties.
 - prometheus
 - nginx
 
-The third parties are installed using `zypper` packages and configured with dedicated roles.
-
-`web` and `wanda` components are installed using containers with the rolling images.
+The third parties are installed using `zypper` packages and configured with dedicated roles. The
+`web` and `wanda` components can be installed using either containers or RPMs. The playbook will
+check the variable `install_method` and depending on the value found (either `docker` or `rpm`) to
+decide which to use.
 
 The `agent` is installed from the configured obs repository using `zypper`.
 
 The nginx configuration acts as a reverse proxy for all the components.
 
 ### SUSE LINUX ENTERPRISE USERS
 
-This playbook assumes you have an activated license of `Suse Linux Enterprise 15 SP5`, with these modules
+This playbook assumes that the server where you are going to install trento-server has an activated license
+of one of the supported OSs, with the modules (Change `x` to match your current version)
 
-- Basesystem Module 15 SP5 x86_64 - `SUSEConnect -p sle-module-basesystem/15.5/x86_64`
-- SUSE Package Hub 15 SP5 x86_64 - ` SUSEConnect -p PackageHub/15.5/x86_64`
-- Containers Module 15 SP5 x86_64 - `SUSEConnect -p sle-module-containers/15.5/x86_64`
+- Basesystem Module 15 x86_64 - `SUSEConnect -p sle-module-basesystem/15.x/x86_64`
+- SUSE Package Hub 15 x86_64 - ` SUSEConnect -p PackageHub/15.x/x86_64`
+- (Optional: for `docker` installation method) Containers Module 15 x86_64 - `SUSEConnect -p sle-module-containers/15.x/x86_64`
 
 ## Usage
 
-You can clone this repository and use them as a normal ansible playbook, making all the updates you need to make to injected variables, embedding other tasks and so on.
+### 1. Clone the repository:
 
-We provide two examples to run the playbook as-is without further modifications.
+`git clone https://github.com/trento-project/ansible.git`
 
-### With local ansible
+### 2. Prepare your inventory file:
 
-- Obtain the playbook, like git cloning this repository
-- Create an [ansible inventory file](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html)
-- Create a json file containing the variables for the playbook or pass them via cli
-- Run provision command
+Get to the `ansible` directory:
+`cd ansible`
 
-**Example inventory to install the trento-server and provision postgres, rabbitmq and prometheus all on the same host**
+Create an `inventory.yml` file, defining the IP address of the machine where each role will be deployed to. You might use the same machine for more
+than one role. Use `;` to comment out any role that you might not want to cover. Here we assume that we have a valid SSH key in `~/.ssh/id_rsa` to
```

Comment:
What does it mean we assume ? 
```Here we assume that we have a valid SSH key in ~/.ssh/id_rsa to access each of the machines.```

Assuming, sounds a bit soft enough for a technical documentation ?

Maybe putting a requirement or must? 
What do you think?